scenes/menus/lobby/Lobby.gd

```diff
@@ -0,0 +1,101 @@
+extends Control
+
+@onready var player_name: LineEdit = $Connect/VBoxContainer2/HBoxContainer/UsernameInput
+@onready var ip_server: LineEdit = $Connect/VBoxContainer2/HBoxContainer2/IPInput
+@onready var host_button: Button = $Connect/VBoxContainer2/HBoxContainer/Host
+@onready var join_button: Button = $Connect/VBoxContainer2/HBoxContainer2/Join
```

Comment:
For the nodes a bit far in the hierarchy, you may prefer to use the unique name instead.